.drone.yml

```diff
@@ -25,72 +25,105 @@ platform:
   arch: amd64
 
 trigger:
+  branch:
+    - main
   event:
     - push
 
 steps:
-  - name: build & push main
+  - name: build & push safe
     image: plugins/docker
+    depends_on:
+      - clone
     settings:
       registry: quay.io
       repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      target: safe
       tags:
         - build_${DRONE_BUILD_NUMBER}
         - ${DRONE_COMMIT_SHA}
```

Comment:
No point in this, my gut feeling is we rely on 'safe' and 'dangerous'. Having this as the default for the SHA isn't totally representative.

Reply:
I thought about forcing a `abcdef-safe` version, but figured that the "safe" version is really just a "base" version that the dangerous version builds on top of, so to me it felt semantically appropriate to have the safe version just be the bare SHA. But I don't mind too much.

(Plus it avoids us referencing "safe" when it's a bit of a misnomer.)

Reply:
I approached it as the `DRONE_COMMIT_SHA` would default be the dangerous imae as its the 'most complete'.